ait/core/server/handler.py

```diff
@@ -125,7 +125,11 @@ def handle(self, input_data):
         Returns:
             tuple of packet UID and packet data field
         """
+
         packet = bytearray(input_data)
+        if (len(packet) < 13):
+            ait.core.log.info('CCSDSPacketHandler: Received packet length is less than minimum of 7 bytes.')
```

Comment:
What's the reason for the discrepancy between checking for length of 12 and saying the minimum is 7? Or am I missing something here?

Reply:
The first 6 bytes are always the same. Possibly extra info coming from ZMQ.

Reply:
Hey @seanlu99, let's dig into this a bit more. I'd like a more concrete answer to what the 6 bytes are, why they're ignored, etc.

Could you post some info on what they are / what you're seeing?